release-notes/1.0/known-issues-1.0.0.md

```diff
@@ -0,0 +1,232 @@
+[runtime.win7-x86.Microsoft.NETCore.Windows.ApiSets](https://dotnet.myget.org/F/dotnet-core/api/v2/package/runtime.win7-x86.Microsoft.NETCore.Windows.ApiSets/1.0.1)
+
+## UCRT Dependency on Windows 7 and System.IO.Compression
+
+With 1.0 of .NET Core, all applications that target .NET Core and run on Windows have a [Universal CRT (UCRT)](https://blogs.msdn.microsoft.com/vcblog/2015/03/03/introducing-the-universal-crt/) dependency. This dependency is needed on all non-Windows 10 editions. This especially impacts the [self-contained](https://dotnet.github.io/docs/core-concepts/app-types.html) applications, as it means that the machine that they are to be run on needs to have UCRT installed.
+
+If the dependency is not present, applications will fail to run and errors will be thrown. The error will be:
+
+> The program can't start because api-ms-win-crt-runtime-1-1-0.dll is missing from your computer. Try reinstalling the program to fix this problem.
+
+The UCRT dependency can be installed via Windows Update (name: "Update for Universal C Runtime in Windows" as according to <https://support.microsoft.com/en-us/kb/2999226>) and as a recommended update it will be installed automatically if the user left the default settings for Windows Update operational. It can also be downloaded from [Microsoft Download Center](https://www.microsoft.com/en-us/download/details.aspx?id=48234). 
+
+
+<https://github.com/dotnet/corefx/issues/9083>
+
+## Bash on Ubuntu on Windows
+
+[Bash on Windows (WSL)](https://msdn.microsoft.com/en-us/commandline/wsl/about) is not yet supported by .NET Core. Attempting to run applications in the environment can experience interittent crashes.
+
+<https://github.com/Microsoft/BashOnWindows/issues/520>
+
+## NegotiateStream.Write interop issues on Linux/OSX
+
+There are some combinations of Kerberos/NTLM, SignOnly/EncryptAndSign etc. that cause the ```gss_wrap``` call on Unix client to fail against a Windows server using ```NegotiateStream```. This issue is for tracking the foll. failing combinations of credentials and protection level passed in ```AuthenticateAsClientAsync```
+
+On Linux:
+
+- Kerberos creds with Sign: Server complains that signature is valid but contents not encrypted
+- NTLM creds with EncrypAndSign: Server rejects signature
+- NTLM creds with Sign: Server rejects signature
+
+On OSX:
+
+- Kerberos creds with Sign: Server complains that signature is valid but contents not encrypted
+- NTLM creds with EncryptAndSign: Server complains about message format
+- NTLM creds with Sign: gss_wrap fails on client side
+
+The fix has been pushed to MIT Kerberos source <https://github.com/krb5/krb5/pull/436> which will make the fix available for all unix platforms, starting krb5-1.15. Red Hat reports the fix will be backported to RHEL 7..
+
+<https://github.com/dotnet/corefx/issues/6767>
+
+## Crypto
+
+Adding certs to an X509Store fails when the current user's home directory is an NTFS volume on Linux/OSX
```

Comment:
This line may or may not want to be promoted to something more titular, since someone could accidentally read that we always don't support NTFS, instead of that we don't support it on Linux and OS X.  (We might also want to normalize the document style for "OS X", "OSX", etc.